### ISSUE TITLE
What the hell?????!!!!!!!!!!

### ISSUE THREAD

**Issue by ForeverYouth** [GeekyAnts/NativeBase#2505]
Hey guys? Are you seriously?!!!!!
What the fuck?!!!!!
I'm been watching your library since v 2.6.1. And each version has new and new icons related problems!!!!!
I do everything that you subscribed in installation guide. But it doesn't work.
I tired open and open new issues! You don't resolve problem!!!! You born new problem!!!
You even can't subscribe normal getting started section without issues!
It's very sad that such stupid Indians make such a good library.

**Comment by danghuyminh** [comment]
Your comment is so negative but I sympathize for the reason you flied off the handle because I feel the same here.

This is my issue: https://github.com/GeekyAnts/NativeBase/issues/2364#issuecomment-430161977

The issue was posted 2 months ago, and I provided a dead simple snippet to reproduce the issue https://snack.expo.io/@yanghuyming/dHJlbW

But all I received was that the issue was closed for duplicated with another issue (I have no idea why they are the same, they are totally different issues).

So I begged them for reopen my issue and check my snippet but they ignored and did nothing. Really disappointed.
